### DISCUSSION TITLE
Theia IDE 1.49 preview testing

### DISCUSSION THREAD

**Discussion by jfaltermeier** [eclipse-theia/theia#13685] [Release Announcements]
The new version 1.49.100 of the Theia IDE is available on the preview channel now, please join the preview testing! You can download it here: [Linux](https://download.eclipse.org/theia/ide-preview/1.49.100/linux/TheiaIDE.AppImage), [Mac](https://download.eclipse.org/theia/ide-preview/1.49.100/macos/TheiaIDE.dmg), [Windows](https://download.eclipse.org/theia/ide-preview/1.49.100/windows/TheiaIDESetup.exe). You can update your existing installation by setting the preference `updates.channel` to preview. Please respond here when you were able to test the preview without finding blockers, by commenting with a ✔️. If you find any issues, please mention them in this thread and report them as an issue once confirmed by other testers.

**Comment by tsmaeder** [comment]
I've used the new version for around 4 hours now and the only thing that I noticed is that once, it looked like the typescript extension did not start when I opened a new window with "Open recent workspace".  Reloading the window fixed it.

**Comment by rschnekenbu** [comment]
No comparison towards a previous version, but I have been using it since this morning and all looks good for me.

**Comment by sgraband** [comment]
I also noticed no regressions. I only noticed that the new splash screen has a white background, which imho looks weird. I opened https://github.com/eclipse-theia/theia/pull/13699 to fix this.

**Reply by tsmaeder** [reply]
I think it looks better this way than with just the icon. Guess there's no accounting for taste 🤷

**Reply by sgraband** [reply]
You can still add your logo with the white background. It's more about what the adopter might expect. If i add a image with transparent background i would expect it to displayed that way.

**Reply by tsmaeder** [reply]
This thread is about Theia IDE, though.

**Comment by tsmaeder** [comment]
Unless there are any objections, I would say we can promote the preview build now. @jfaltermeier would you do the honors?

**Reply by jfaltermeier** [reply]
The preview got promoted now

**Comment by jfaltermeier** [comment]
We've published a new preview 1.49.101
This adds missing window icons on Linux and updates dependencies to their latest Versions. Theia version and Built-ins are unchanged, so the overall functionality should be unchanged. 
Please let us know if you find any issues

**Reply by JonasHelming** [reply]
New version can be downloaded here: https://download.eclipse.org/theia/ide-preview/1.49.101/ (or of course via the automatic update)

**Reply by tsmaeder** [reply]
I am seeing delays when giving focus to another application and then back to Theia IDE today. The IDE is unresponsive for a couple of seconds and the Task Manager shows a lot of CPU for the Theia renderer process briefly. I had not noticed this yesterday. Anyone else have this?
